lib/upload.ex

```diff
@@ -157,7 +163,7 @@ defmodule Upload do
   """
   @spec generate_key(String.t(), [{:prefix, list}]) :: String.t()
   def generate_key(filename, opts \\ []) when is_binary(filename) do
-    if Keyword.get(opts, :generate_key) do
+    if Keyword.get(opts, :generate_key, true) do
```

Comment:
We were always generating hashed keys for uploads as the default was `nil` here.

Reply:
So we _weren't_ generating these UUID prefixes for filenames, right? `nil` is falsey, so changing the default to `true` inverts which clause is executed here by default (changing it to the branch that generates the keys).

Reply:
Right, I misspoke, we **want** to hash by default as that's what the library intends. We always specify `generate_key: false` in Joydrive.